### DISCUSSION TITLE
Selected Mods(+/-) leaderboard that show all mod combinations that include the selected mods

### DISCUSSION THREAD

**Discussion by ashandelle** [ppy/osu#29674] [Ideas]
~~When using the Selected Mods option the leaderboard only shows scores with the exact same mods, if you have HR selected you only see HR in the leaderboard. I think there should be another (separate) option that shows scores including selected mods, so if you have HR selected you will see HDHR scores too.~~

Right now there is only one option to change how the leaderboard displays mod combinations, Selected Mods, which only shows other plays with the exact same mod combinations. I think there should be 2 more **additional** options, Selected Mods+ and Selected Mods-.

Selected Mods+ will display all mod combinations including the one selected, so if you have the mod HR selected then the leaderboard will show HR, HDHR, HRNF and any other combination including HR.

Selected Mods- will display all mod combinations that are included within the one selected, so if you have the mod HDHR selected then the leaderboard will show HDHR, HD, HR, NM and nothing else.

**Comment by MizuIro-H** [comment]


**Reply by ashandelle** [reply]
I thought I made it clear that I want it to be a separate option.
HR and HDHR was simply an example to show how it works. I don't think making a distinction for HR and HDHR is a good idea as it complicates an option that would otherwise be a simple extension of the existing one.
My actual reason for suggesting this is that I frequently play with ST, but there are very few other players that do too, and some of the ones that do play with HD, TC, NF, etc, and this option would allow me to see the scores anyway.

**Reply by MizuIro-H** [reply]
My bad and I  apologize for my misread.
I'll delete it soon.

**Comment by MizuIro-H** [comment]
I support this as I play mania often with AC and MU. since these don't make a large difference with NM players, I'm happy to have a option to let me compare my AC score with NM players.

**Reply by ashandelle** [reply]
I don't see how that would work with the original proposal as NM doesn't include AC which is the point of the option, but you have brought up a good point so ill add it as another option.

**Comment by Wypixla1** [comment]
I don't see a use case for Selected Mods- but SelectedMods+ would actually be pretty useful, especially in lazer where there are a bunch of fun mods
